CHANGES.rst

```diff
@@ -10,6 +10,42 @@
 
 .. towncrier release notes start
 
```

Comment:
FWIW, it seems like the previous changelog updates had two empty lines as a separator. Dunno if we want it that way, though.
```suggestion


```

Reply:
Once you've merged the updated template, I'll just update the existing changelog to follow whatever towncrier produces. But, this is what it produced when I ran it today.